anaconda.spec.in

```diff
@@ -10,7 +10,7 @@ URL:     http://fedoraproject.org/wiki/Anaconda
 # git checkout -b archive-branch anaconda-%%{version}-%%{release}
 # ./autogen.sh
 # make dist
-Source0: %{name}-%{version}.tar.bz2
+Source0: https://github.com/rhinstaller/%{name}/releases/download/%{name}-%{version}-1/%{name}-%{version}.tar.bz2
```

Comment:
Why the `-1`? You don't do tags that way...

Reply:
Also, the rules state you should only do this if you're _actually_ uploading tarballs there. The URLs need to be valid.

Reply:
I will not adjust this in this PR, as it's just a backport. I will discuss today with the team about changing the versioning practice, thanks for raising.

Reply:
Might have some time to handle this week https://issues.redhat.com/browse/INSTALLER-3953 

Some more  infra will needs adjustments, therefore not doing it directly.

Reply:
I think we have/had the `-1` in many places and indeed, its more or less wrong. It's just that I'm afraid it might be hardcoded in quite a few places in our infra to just remove everywhere at once. :P

Reply:
The tag should just be the version, rather than `name-version-1`. If we can fix that, then it will also make it easier for other distributions shipping Anaconda to track it through things like release-monitoring and whatnot.